src/v/pandaproxy/schema_registry/store.h

```diff
@@ -705,6 +713,55 @@ class store {
         return outcome::success();
     }
 
+    void setup_metrics() {
+        namespace sm = ss::metrics;
+        auto setup_common = [this]<typename MetricDef>() {
+            std::vector<MetricDef> defs;
+            defs.reserve(3);
+            defs.emplace_back(
+              sm::make_gauge(
+                "schema_count",
+                [this] { return _schemas.size(); },
+                sm::description("The number of schemas in the store"))
+                .aggregate({}));
```

Comment:
what does the `aggregate` with an empty argument list do?
Is it different than not calling it at all?

Reply:
> what does the `aggregate` with an empty argument list do? Is it different than not calling it at all?

Mostly to remind me that I'm not aggregating anything, and in particular, not `shard_label`.